src/lib/plugin.js

```diff
@@ -236,6 +236,7 @@ class FiveBellsLedger extends EventEmitter2 {
       this.connection = reconnect({immediate: true}, (ws) => {
         ws.on('open', () => {
           this.connected = true
+          this.emit('connect')
```

Comment:
I could change it to `if (!connected) { this.emit('connect') } this.connected = true`, in another PR. Would that fix it?